packages/xrpl/src/models/methods/index.ts

```diff
@@ -248,6 +254,120 @@ type Response =
   | NFTInfoResponse
   | NFTHistoryResponse
 
+export type RequestResponseMap<T> = T extends AccountChannelsRequest
+  ? AccountChannelsResponse
+  : T extends AccountCurrenciesRequest
+  ? AccountCurrenciesResponse
+  : T extends AccountInfoRequest
+  ? AccountInfoResponse
+  : T extends AccountLinesRequest
+  ? AccountLinesResponse
+  : T extends AccountNFTsRequest
+  ? AccountNFTsResponse
+  : T extends AccountObjectsRequest
+  ? AccountObjectsResponse
+  : T extends AccountOffersRequest
+  ? AccountOffersResponse
+  : T extends AccountTxRequest
+  ? AccountTxResponse
+  : T extends GatewayBalancesRequest
+  ? GatewayBalancesResponse
+  : T extends NoRippleCheckRequest
+  ? NoRippleCheckResponse
+  : // NOTE: The order of these LedgerRequest types is important
+  // to get the proper type matching overrides based on parameters set
+  // in the request.
```

Comment:
Is there documentation this comment could link to which helps inform what the proper order should be? (Could be useful if we have to add another one "in the right spot" later on.

Reply:
Added docs inline, but they're a bit hefty